server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/ProjectTester.java

```diff
@@ -43,25 +43,26 @@
   }
 
   void deleteAll() {
+    org.sonarqube.ws.client.project.ProjectsService serviceOld = session.wsClient().projectsOld();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "ProjectsService"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "projectsOld"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)